torch/_inductor/ir.py

```diff
@@ -6682,7 +6682,10 @@ def tensor_to_layout(output: torch.Tensor):  # type: ignore[no-untyped-def]
 
     @classmethod
     def create(cls, kernel, *args, **kwargs):  # type: ignore[no-untyped-def]
-        fake_incorrect_kernels = (aten._fused_moving_avg_obs_fq_helper_functional,)
+        fake_incorrect_kernels = (
```

Comment:
This doesn't work with symints, would it be possible to fix the meta instead ?

Reply:
To fix the meta, we should duplicate [this C++ function]( https://github.com/pytorch/pytorch/blob/6d56277682715e56cfdfcaff6f770acebda966d7/aten/src/ATen/TensorIterator.cpp#L232-L308) into torch/_meta_implementations.py right?

Reply:
I feel we should already have reorder_dimensions but I do not see it. Check if `compute_elementwise_output_logical_to_physical_perm` is what you want.